docs/package/gluon-mesh-batman-adv.rst

```diff
@@ -0,0 +1,202 @@
+gluon-mesh-batman-adv
+=====================
+
+.. image:: gluon-mesh-batman-adv-logo.svg
+   :width: 300 px
+
+B.A.T.M.A.N. Advanced (often referenced as batman-adv) is an implementation of
+the B.A.T.M.A.N. routing protocol in form of a linux kernel module operating on layer 2.
+
+Layer 2 means that all client devices will operate in the same, virtual broadcast
+domain and will see each other "as if they were connected to one giant switch".
+
+This comes with a set of advantages (like quick and economical client device roaming,
+layer 3 protocol agnosticism, broadcast/multicast). But also impediments, especially
+layer 2 multicast overhead - which Gluon tries to mitigate to achieve a certain degree
+of scalability. See :doc:`gluon-ebtables-filter-multicast` and
+:ref:`batman-adv-multicast-architecture` for details.
+
+B.A.T.M.A.N. Advanced project homepage:
+
+* https://www.open-mesh.org/projects/batman-adv/wiki/Wiki
+
+Flavours
+--------
+
+Gluon currently supports two main build flavours of batman-adv:
+
+gluon-mesh-batman-adv-15
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is the recommended batman-adv flavour to use.
+
+It follows recent, upstream batman-adv releases and is flexible to new feature additions.
+
+gluon-mesh-batman-adv-14 (`batman-adv-legacy`)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`gluon-mesh-batman-adv-14`, also known as `batman-adv-legacy` or batman-adv v2013.4
+is the last batman-adv release with the batman-adv compatibility version 14, which
+was released in October 2013.
+
+With batman-adv v2014.0.0 a compat breakage became necessary for the introduction
+of new features. However, one of these features was the addition of TVLV support
+(type-version-length-value fields) which from then on allowed adding features
+without breaking packet format compatibility. This made it possible to stay with
+compatibility version 15 so far.
+
+For new installations `gluon-mesh-batman-adv-14` is **not recommended**. It misses
+a lot of bugfixes and is currently only available for existing communities
+until they have migrated. This package will soon be deprecated and removed.
```

Comment:
minor suggestion: maybe also mention troubles when building old batmans or new kernels to give this even more punch?

Reply:
Thanks for your review and comment! Just to let you know, I haven't added this because we actually maintained and backported patches fixing build issues on newer kernels from upstream batman-adv to batman-adv-legacy so far. So building batman-adv-legacy on recent kernels shouldn't be an issue.